lib/template.html

```diff
@@ -35,6 +35,20 @@
   {% assign seo_description = seo_description | markdownify | strip_html | strip_newlines | escape_once %}
 {% endif %}
 
+{% if page.author %}
+  {% if page.author.name %}
+    {% assign seo_author_name = page.author.name %}
+  {% else %}
+    {% assign seo_author_name = page.author %}
+  {% endif %}
+
+  {% if page.author.twitter %}
+    {% assign seo_author_twitter = page.author.twitter | replace:"@","" | prepend:"@" %}
+  {% else %}
+    {% assign seo_author_twitter = page.author | replace:"@","" | prepend:"@" %}
```

Comment:
1. It looks like you're adding the `@`? I believe the Twitter spec says it shouldn't have an `@`, but I haven't looked at it in a while. 

2. We can DRY things up a bit by moving the normalization down to the one place it's used.